ci.no

```diff
@@ -0,0 +1,4 @@
+#/bin/sh
+
+# Simulate the result of running a typical Jenkins pipeline.
```

Comment:
This should have a `sleep 9000` to get an authentic Jenkins experience

Reply:
Ugh, you're right.